ha/core/cluster/cluster_manager.py

```diff
@@ -283,6 +283,9 @@ def __init__(self):
             # Example: cm.cluster_controller.start()
             # Find more example in test case.
             self.__dict__[controller] = self._controllers[controller]
+        for controller in self._controllers.keys():
```

Comment:
Can we move this in ElementControllerFactory
as it is part of class build

Reply:
Moved